demisto_sdk/commands/test_content/TestContentClasses.py

```diff
@@ -513,7 +514,9 @@ def _get_unmockable_tests_from_conf(self) -> list:
                                             integration_name in unmockable_integrations]
             if test_name and (not test_record.test_integrations or unmockable_integrations_used) or not self.isAMI:
                 unmockable_tests.append(test_record)
-
+                continue
```

Comment:
why do we continue here?

Reply:
In case a test has both - an unmockable integration and is configured with is_mockable=False - it will be added twice if we don't continue.

Reply:
please add a comment for that here to explain that